apps/stripe/src/modules/saleor/saleor-webhook-responses.ts

```diff
@@ -0,0 +1,95 @@
+import { GetConfigError, MissingConfigError } from "@/modules/app-config/app-config-errors";
+import { SaleorApiUrl } from "@/modules/saleor/saleor-api-url";
+
+export class SuccessWebhookResponse {
+  statusCode = 200;
+}
+
+export class ErrorWebhookResponse {
+  statusCode = 500;
+}
+
+export class GetConfigErrorResponse extends ErrorWebhookResponse {
+  error: InstanceType<typeof GetConfigError>;
+  message = "App is not configured - error while getting config";
+
+  constructor(args: { error: InstanceType<typeof GetConfigError> }) {
+    super();
+    this.error = args.error;
+  }
+
+  getResponse() {
+    return Response.json(
+      {
+        message: this.message,
+      },
+      { status: this.statusCode },
+    );
+  }
+}
+
+export class MissingConfigErrorResponse extends ErrorWebhookResponse {
+  error: InstanceType<typeof MissingConfigError>;
+  message = "App is not configured - configuration is missing for channel";
+
+  constructor(args: { error: InstanceType<typeof MissingConfigError> }) {
+    super();
+    this.error = args.error;
+  }
+
+  getResponse() {
+    return Response.json(
+      {
+        message: this.message,
+        channelId: this.error.channelId,
```

Comment:
is Saleor going to actually read channelId?

Reply:
No idea - I will remove it then from from json response